css/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+	box-sizing: border-box;
+	font-family: "Times New Roman", Times, serif;
+}
+
+h1 {
+	margin-bottom: 40px;
+	text-align: center;
+	font-weight: bold;
+	font-family: 'Alice', serif;
+}
+
+section {
+	position: relative;
+	font-weight: bold;
+	}
+
+#boxchicken {
+	background-color: #778899;
+	font-family: 'Alice', serif;
+	position: absolute;
+	border-style: solid;
+	border-width: 1px;
+	font-size: 120%;
+	padding: 5px 30px 5px 30px;
+	top: 0;
+	right: 0;
+	}	
```

Comment:
For all 3 of the boxes the following rules can be moved into a class to prevent repeating: font-family, position, border-style, border-width, font-size, padding, top, right.

Reply:
Thank you, change made. Truly helpful tips!